cache.go

```diff
@@ -483,3 +483,35 @@ func (s *SimpleCache) Fetch(id uint64) (*Bitmap, bool) {
 func (s *SimpleCache) Add(id uint64, b *Bitmap) {
 	s.cache[id] = b
 }
+
+type NopCache struct {
+	stats StatsClient
+}
+
+// NopCache implement Cache interface, returns no cache for cache type None
```

Comment:
Let's change this to be consistent with the other Cache implementation comments:
```
// Ensure NopCache implements Cache.
```